db/migrate/20230601112552_create_line_items.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLineItems < ActiveRecord::Migration[7.0]
+  def change
+    create_table :line_items do |t|
+      t.integer :quantity, default: 0
+      t.references :order, null: false, foreign_key: true
+      t.references :meal, null: false, foreign_key: true
```

Comment:
Add line item total here as well as meal price can be changed later, but we should have the price at which item was ordered

Reply:
Done